hack/build-e2e-race.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e
+
+BUILD_DIR="${BUILDDIR:=test}"
+SRC_DIR="${SRCDIR:=.}"
+
+# Create directory if it doesn't exist
+if [ ! -d $BUILD_DIR ]
+then
+    mkdir ./$BUILD_DIR
+fi
+
+CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build -race -ldflags "-s -w" -o $BUILD_DIR/devpod-linux-amd64 $SRC_DIR
```

Comment:
same as below

Reply:
this was something I was trying so we can use the race detector in the CI. I've removed now as I think this will take some time to get working in the CI